src/providers/export/azureCustomVision.test.ts

```diff
@@ -68,7 +72,10 @@ describe("Azure Custom Vision Export Provider", () => {
         };
 
         testProject.exportFormat.providerOptions = customVisionOptions;
-        const provider = new AzureCustomVisionProvider(testProject, testProject.exportFormat.providerOptions);
+        const provider = new AzureCustomVisionProvider(
+            testProject,
+            testProject.exportFormat.providerOptions as IAzureCustomVisionExportOptions,
```

Comment:
why pass `providerOptions` when we're already passing in the` testProject`?

Reply:
This is an artifact of the `IExportProvider` implementation.  It requires the project to export as well as the options required for the export provider.  In this case it is part of the project settings itself.  I'm not super happy about this either but didn't want to introduce a change here.